src/Uno.UI/UI/Xaml/Media/ImageSource.Android.cs

```diff
@@ -219,13 +219,13 @@ internal async Task<Bitmap> Open(CancellationToken ct, Android.Widget.ImageView
 
 			if (FilePath.HasValue())
 			{
-				await BitmapFactory.DecodeFileAsync(FilePath, options);
+				BitmapFactory.DecodeFile(FilePath, options);
```

Comment:
This will cause the UI to freeze when loading images. Why do you need the synchronous version?